patches/components-update_client-pipeline.cc.patch

```diff
@@ -0,0 +1,23 @@
+diff --git a/components/update_client/pipeline.cc b/components/update_client/pipeline.cc
+index 9e922a09e674ea89a1e84832468595cc84d9d9a5..c60f89f3f34fd6d58fd552303f88db7585d96f91 100644
+--- a/components/update_client/pipeline.cc
++++ b/components/update_client/pipeline.cc
+@@ -77,9 +77,7 @@ constexpr CategorizedError kInvalidOperationAttributesError = CategorizedError(
+ // completion callback will keep it alive).
+ class Pipeline : public base::RefCountedThreadSafe<Pipeline> {
+  public:
+-  Pipeline(
```

Comment:
what is this change?

Reply:
We need to update our go-updater to supply `size` attribute. This is a temporary workaround while that work is under way.